lib/ansible/executor/module_common.py

```diff
@@ -281,16 +286,14 @@ def debug(command, zipped_mod, json_params):
             with open(args_path, 'rb') as f:
                 json_params = f.read()
 
-            # Monkeypatch the parameters into basic
-            from ansible.module_utils import basic
-            basic._ANSIBLE_ARGS = json_params
+            from ansible.module_utils._internal._ansiballz import run_module
 
-            # Run the module!  By importing it as '__main__', it thinks it is executing as a script
-            runpy.run_module(mod_name=%(module_fqn)r, init_globals=None, run_name='__main__', alter_sys=True)
-
-            # Ansible modules must exit themselves
-            print('{"msg": "New-style module did not handle its own exit", "failed": true}')
-            sys.exit(1)
+            run_module(
+                json_params=json_params,
+                profile=%(profile)r,
+                module_fqn=%(module_fqn)r,
+                modlib_path=modlib_path,
```

Comment:
```suggestion
                modlib_path=zipped_mod,
```

`modlib_path` isn't defined in this method. `zipped_mod` is the value passed in.